resources.go

```diff
@@ -2209,6 +2209,55 @@ func Provider() tfbridge.ProviderInfo {
 				"Pulumi":                       "1.5.0-*",
 				"System.Collections.Immutable": "1.6.0",
 			},
+			Namespaces: map[string]string{
+				"appsync":           "AppSync",
+				"appmesh":           "AppMesh",
+				"apigateway":        "ApiGateway",
+				"appautoscaling":    "AppAutoScaling",
+				"autoscaling":       "AutoScaling",
+				"cloudformation":    "CloudFormation",
+				"cloudhsmv2":        "CloudHsmV2",
+				"cloudfront":        "CloudFront",
+				"cloudtrail":        "CloudTrail",
+				"cloudwatch":        "CloudWatch",
+				"codebuild":         "CodeBuild",
+				"codecommit":        "CodeCommit",
+				"codedeploy":        "CodeDeploy",
+				"codepipeline":      "CodePipeline",
+				"datapipeline":      "DataPipeline",
+				"datasync":          "DataSync",
+				"devicefarm":        "DeviceFarm",
+				"directoryservice":  "DirectoryService",
+				"docdb":             "DocDB",
+				"dynamodb":          "DynamoDB",
+				"directconnect":     "DirectConnect",
+				"ec2clientvpn":      "Ec2ClientVpn",
+				"ec2transitgateway": "Ec2TransitGateway",
```

Comment:
Ec2 or EC2.  .net generally says "if it's two letters capitalize both.  if its 3+ then title case". i don't know where EC2 lands.